autofit/non_linear/search/abstract_search.py

```diff
@@ -597,6 +599,14 @@ def pre_fit_output(
                 model=model,
             )
 
+            timeout_seconds = get_timeout_seconds()
+
+            if timeout_seconds is not None:
+
+                logger.info(f"\n\n ***Log Likelihood Function timeout is "
+                            f"turned on and set to {timeout_seconds} seconds.***\n")
+

```

Comment:
Doesn't do anything yet?

Reply:
It prints that the time out feature is turned on.

I couldnt do this in `fitness.py` cause it needs to be wtihin the `is_master` parallel calls.

Reply:
Ah makes sense